dev/provider_packages/refactor_provider_packages.py

```diff
@@ -142,6 +142,7 @@ def rename_deprecated_modules(self) -> None:
             ("airflow.operators.bash", "airflow.operators.bash_operator"),
             ("airflow.operators.python", "airflow.operators.python_operator"),
             ("airflow.utils.session", "airflow.utils.db"),
+            ("airflow.providers.cncf.kubernetes.backcompat", "airflow.kubernetes"),
```

Comment:
Oh - I just saw this -- that'll help us for backport, but won't help for anyone trying to use it on 2.0.0 (i.e. upgrade then fix errors approach)